stackslib/src/chainstate/stacks/mod.rs

```diff
@@ -627,10 +631,8 @@ impl_byte_array_serde!(TokenTransferMemo);
 pub enum TenureChangeCause {
     /// A valid winning block-commit
     BlockFound = 0,
-    /// No winning block-commits
-    NoBlockFound = 1,
-    /// A null miner won the block-commit
-    NullMiner = 2,
+    /// The next burnchain block is taking too long, so extend the runtime budget
+    Extended = 1,
 }
```

Comment:
It looks like this implementation differs from what is currently described in [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md) -- rather than two distinct tx types `TenureChange` and `TenureExtension`, this PR bundles the functionality of both into one tx type. Is this an intended change?

Reply:
Yes, it's intentional.  `TenureExtend` is no longer a transaction type; instead there are two different variants of `TenureChange`.